src/controller/connexionController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace TourCMS\OnBoarding\Controller;
+
+use Error;
+use TourCMS\OnBoarding\Model\connexion;
+
+class connexionController
+{
+    protected $connexion;
```

Comment:
```suggestion
    protected $connection;
```
Try to use english vars